### PR TITLE
Placing the app platform tutorials where they belong in the menu

### PR DESCRIPTION
**Pull request by marians**
Preview

image

### CODE REVIEW

File: src/content/tutorials/fleet-management/app-platform/_index.md [hunk 1]
```diff
@@ -1,11 +1,10 @@
 ---
 linkTitle: App platform
-title: Guides for using the App platform
+title: App management
```
Comment (AndiDog) [inline src/content/tutorials/fleet-management/app-platform/_index.md:3]: It's strange to see a different title in menu vs. on the right side. This page doesn't describe the "App platform".

Suggestion: only `title: App management`

Comment (marians) [reply src/content/tutorials/fleet-management/app-platform/_index.md:3]: You're right, will fix

**Comment by marians** [comment]
@AndiDog Fine to merge?

**Review by lyind — APPROVED** [review]
`tutorials-fleet-management` seems to be the correct parent.

LGTM

**Pull request merged**